docs/api.rst

```diff
@@ -556,7 +556,15 @@ Extended handlers
     When True requires SSL/TLS to be established on the data channel. This
     means the user will have to issue PROT before PASV or PORT (default
     ``False``).
+	
+  .. data:: client_certfile
 
+    The path of the certificate to check the client certificate against. 
+    When provided, only allowing clients with a valid certificate to connect 
+    to the server (default ``None``).
+
+     .. versionadded:: 1.5.3    
+	
```

Comment:
This is not entirely clear. To be cristal clear, does it mean the client must have the exact same certificate? If so, I would recommend this wording:

    The path to a file which contains a certificate to be used to 
    identify the client. If specified, the client should is supposed to
    use the same certificate in order to connect.